app/models/household.rb

```diff
@@ -1,16 +1,27 @@
 class Household < ApplicationRecord
   has_many :children
+  before_save :set_huid
```

Comment:
Either way can work; I just tend to put stuff like this into the model, since it's fundamentally a data concern, while I tend to put stuff that directly deals with user interaction into the controller.

Reply:
I'd prefer the logic to save this field go in the SignatureForm since that's the pattern already established. Checking for state in the `set_huid` private method creates an overcomplicated dependency.

Reply:
Done